humans_num: 3
robots_num: 6
seed: 0
prob: 0.1
dt: 0.5
horizon: 10
safety_dist: 1.0
area_width: 20.0
obstacles: [[-4.5, 4.5], [-3.5, -4.5], [2.5, -2.5], [4.5, 2.5], [5.5, -6.5], [-6.5, -2.5], [1.5, 7.5]]
covariances:
- - - 5.008405965116535
    - 0.03454986400504001
  - - 0.016677634653232142
    - 5.087855908553266
- - - 5.095096403162672
    - 0.0038748375832319826
  - - 0.0699107391398761
    - 5.057275981556941
- - - 5.089800711822515
    - 0.06668989729196366
  - - 0.05478377830464816
    - 5.070242742406806
- - - 5.038647716961466
    - 0.06944386777644583
  - - 0.08248704645005482
    - 5.0465668475367265
means:
- - -9.792516922286001
  - 0.03749184297477548
- - -0.08453413731707826
  - -7.323409420814468
- - -7.15777829090923
  - -5.6288264874784355
- - -1.6298363897909507
  - -5.037976631728817
weights:
- 0.25
- 0.25
- 0.25
- 0.25
